### ISSUE TITLE
Auto config does not work on jumping between networks

### ISSUE THREAD

**Issue by fwenzel** [synergy/synergy#4408]
Use case:
* 1 Synergy server at home, 1 at the office, both configured to accept the same client (a laptop)
* auto config should recognize a network change (i.e., opening the laptop in either location) and connect to the respectively available server there.

Sadly, this does not happen. Instead, I have to uncheck and re-check auto config, or uncheck it and enter the right server name in the settings.

**Comment by fwenzel** [comment]
My crystal ball debugging skills tell me, that perhaps the client does not realize it should re-detect servers via Bonjour. So it keeps trying to reconnect to the last known server, instead of the only available one.

**Comment by nbolton** [comment]
Thanks for raising this @fwenzel. I think you're right... I think remember @XinyuHou closing and re-opening the GUI a lot when he was testing -- not a good sign ;-)

**Comment by XinyuHou** [comment]
@fwenzel You are right. That is what Synergy is doing right now. We need to improve it in 1.7.1.

One solution would be check if network has changed after certain amount of connection failure.
